packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -1216,6 +1213,7 @@ export class MgtPeoplePicker extends MgtTemplatedTaskComponent {
   protected handleRemovePersonKeyDown(person: IDynamicPerson, e: KeyboardEvent): void {
     if (e.key === 'Enter') {
       this.removePerson(person, e);
+      this.enableTextInput();
```

Comment:
Blocking comment to prevent merging until we have a11y confirmation that this fixes the issue

Reply:
@gavinbarron we have a11y confirmation. Resolving...